### PR TITLE
Updated zilliqa readme

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@junhaoyap please ignore the other files and only look at the README.md in the Zilliqa repo. 

I forked this branch from the other branch (let's call this branch A) but for some reason after merging branch A, it still shows all the other files that were modified in branch A.

**Comment by junhaoyap** [comment]
Huh ya strange ah, ok will focus on README

File: Zilliqa/README.md [hunk 1]
```diff
@@ -11,46 +11,42 @@ https://www.xfers.com/
 The XSGD contract has been written by the StraitsX team to fit both the specific needs of the XSGD token as described in the StraitsX Whitepaper and the requirement to comply with local regulation.
 
 The XSGD contract consists of two communicating contracts:
-- [token contract](https://github.com/Xfers/XSGD-scilla/blob/master/contracts/xsgd_contract.scilla)
-- [proxy contract](https://github.com/Xfers/XSGD-scilla/blob/master/contracts/proxy.scilla)
+- [token contract](https://github.com/Xfers/StraitsX-tokens/blob/master/Zilliqa/contracts/xsgd_contract.scilla)
```
Comment (junhaoyap) [inline Zilliqa/README.md:14]: Ohh yeah these are very outdated

File: Zilliqa/README.md [hunk 2]
```diff
@@ -121,29 +116,34 @@ Proxy contract is a relay contract that redirects calls to it to the token contr
 
 | Name | Description & Privileges |
 |--|--|
-|`init_admin` | The initial admin of the contract which is usually the creator of the contract.  `init_admin` is also the initial value of `admin`. |
-|`admin` | Current `admin` of the contract initialized to `init_admin`. Certain critical actions can only be performed by the `admin`, e.g., changing the current implementation of the token contract. |
-|`initiator`| The user who calls the proxy contract that in turns call the token contract. After deployment, the address of the proxy contract will be made known to the user and the code will be visible directly from the block explorer. |
+|`init_admin`| The initial admin of the contract. It is usually the creator of the contract. |
```
Comment (junhaoyap) [inline Zilliqa/README.md]: Description of admin seems to have disappeared, i.e. changing implementation of contract

File: Zilliqa/README.md [hunk 3]
```diff
@@ -121,29 +116,34 @@ Proxy contract is a relay contract that redirects calls to it to the token contr
...
+The table below presents the mutable fields of the token contract and their initial values.
 
 | Name | Type | Initial Value |Description |
 |--|--|--|--|
-|`implementation`| `ByStr20` | `init_implementation` | Address of the current implementation of the token contract. |
-|`admin`| `ByStr20` | `init_owner` | Current `admin` in the contract. |
-|`balances`| `Map ByStr20 Uint128` | `Emp ByStr20 Uint128` | Keeps track of the number of tokens that each token holder owns. |
-|`totalSupply`| `Uint128` | `0` | The total number of tokens that is in the supply. |
+|`implementation`|`ByStr20`|`init_implementation` | Current `implementation` address of the token contract. |
+|`admin`|`ByStr20`|`init_admin` | Current `admin` address of the token contract. |
```
Comment (junhaoyap) [inline Zilliqa/README.md:144]: Also not really described here

**Review by junhaoyap — APPROVED** [review]
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FxT9DPrC1YWXBO1coyQ%2Fgiphy.gif&token=lgtm" data-gitmeme-token="lgtm"></a>

One minor feedback to consider addressing

File: Zilliqa/README.md (hunk 2 above)
Comment (ghost) [reply Zilliqa/README.md]: done, I think I might have accidentally taken it out.

**Pull request merged**